docs/docs/releases/0.2.0.md

```diff
@@ -481,137 +482,120 @@ We will assume you're using our provided`docker-compose.yml` file. If you have h
+    - The default values for them are `mathesar` and `mathesar`.
+    - If you have any external hosts (other than `mathesar_db`), you should probably have the postgres user and passwords with you.
+
 ### For installations done from scratch
 
 For these instructions, you'll need to be familiar enough with your setup to log into your internal Mathesar Database.
 
 1. Log into your `mathesar_django` database, and run:
-
     ```sql
-    CREATE DATABASE mathesar_temp;
+    SELECT db_name,host,port FROM mathesar_database;
     ```
+    - The above SQL query will provide the list of connected database names along with their hosts and port.
+    - Copy and paste them in a text file somewhere. We will need them after installing 0.2.0.
 
-1. Log out of the database, and log into the `mathesar_temp` database. Then run:
+1. Now, run the following:
 
     ```sql
-    DROP DATABASE mathesar_django;
-    CREATE DATABASE mathesar_django OWNER mathesar;  -- or whichever role you use to run Mathesar
-    ```
-
-1. Log out of the database, and log into your new `mathesar_django` database. Then run:
-
-    ```sql
-    DROP DATABASE mathesar_temp;
-    ```
-
-1. Go to your Mathesar installation directory
-
-    ```
-    cd xMATHESAR_INSTALLATION_DIRx
-    ```
-
-    !!! note
-        Your installation directory may be different from above if you used a different directory when installing Mathesar.
-
-1. Pull version 0.2.0 from the repository
-
-    ```
-    git pull https://github.com/mathesar-foundation/mathesar.git
-    git checkout 0.2.0
-    ```
-
-1. Update Python dependencies
-
-    ```
-    pip install -r requirements.txt
-    ```
-
-1. Activate our virtual environment
-
-    ```
-    source ./mathesar-venv/bin/activate
-    ```
+    -- Create a temporary database
+    CREATE DATABASE mathesar_temp;
 
-1. You can skip the following if you're upgrading from versions 0.1.4 and above.
-    - If you're upgrading from versions <= 0.1.3, update your environment variables according to the [the new configuration specification](../administration/configuration.md#db).
-    - In particular, you must put the connection info for the internal DB into new `POSTGRES_*` variables. The `DJANGO_DATABASE_URL` variable is no longer supported.
+    -- Change current database to the temporary database
+    \c mathesar_temp;
 
-1. Add the environment variables to the shell before running Django commands
+    -- Drop the old mathesar_django database
+    DROP DATABASE mathesar_django;
 
-    ```
-    export $(sudo cat .env)
-    ```
+    -- Create a new mathesar_django database to use with 0.2.0
+    -- Set owner to whichever role you use to run Mathesar
+    CREATE DATABASE mathesar_django OWNER mathesar;
 
-1. Run Django migrations
+    -- Change current database to the new mathesar_django database
+    \c mathesar_django;
 
-    ```
-    python manage.py migrate
+    -- Drop the temporary database
+    DROP DATABASE mathesar_temp;
```

Comment:
I didn't check in the "install from scratch" case, but I suspect this will break there too.

Reply:
Modified there as well.